tests/benchdnn/inputs/reorder/harness_reorder_compensation

```diff
@@ -51,7 +51,8 @@
 --attr-scales=,src:common:2
 --oflag=,s8s8_comp:3,zp_comp:3,s8s8_comp:3+zp_comp:3
 --stag=abx,xcab
---dtag=xcab        2x32x32x3 2x32x32x3x3 16x32x32x3x3x3
+--dtag=xab,xcab
+2x32x32x3 2x32x32x3x3 16x32x32x3x3x3 79x79x2
```

Comment:
`xcab` tips this is a grouped-channels case. `c` is input channels, `a` is groups of output channels, `b` is output channels.

For `xab`, or non-grouped cases, I would really suggest to create a new line:
```
--dtag=xcab 2x32x32x3 2x32x32x3x3 16x32x32x3x3x3
--dtag=xab 79x79x2
```
Otherwise, I'd expect either errors or some irrelevant test cases for grouped-channels scenario.
BTW, splitting is always safer and more explicit with a price of extra typing.